### PR TITLE
Add EWS example

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[2385a1e](https://github.com/OfficeDev/outlook-dev-docs/pull/465/commits/2385a1e5ae94b9f4aaf9a40c9c85a9cc64ad26cb)_: 

:clock10: **Preparing**: average preparing time is _41 sec(s)_

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[2385a1e](https://github.com/OfficeDev/outlook-dev-docs/pull/465/commits/2385a1e5ae94b9f4aaf9a40c9c85a9cc64ad26cb)_: 

:clock10: **Incremental building**: average incremental building time is _17 sec(s)_

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[2385a1e](https://github.com/OfficeDev/outlook-dev-docs/pull/465/commits/2385a1e5ae94b9f4aaf9a40c9c85a9cc64ad26cb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/metadata-for-an-outlook-add-in.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-add-ews-example/docs/add-ins/metadata-for-an-outlook-add-in.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C19%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201812192358243653-465%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=iiooqqLEBTVm0nUrA%2BgqtKVSJT2w1cQWHAA50qOBvoc%3D&st=2018-12-19T23%3A54%3A27Z&se=2019-01-19T23%3A59%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/add-ins/metadata-for-an-outlook-add-in.md [hunk 1]
```diff
@@ -201,6 +201,57 @@ Custom properties set by an add-in are not equivalent to normal MAPI-based prope
...
+Office.context.mailbox.makeEwsRequestAsync(
+    request_str,
+    function(asyncResult) {
+        if (asyncResult.status === Office.AsyncResultStatus.Succeeded) {
+            console.log(asyncResult.value);
+        }
+        else {
+            console.log(JSON.stringify(asyncResult));
+        });
+```
```
Comment (kbrandl) [inline docs/add-ins/metadata-for-an-outlook-add-in.md]: Code sample is missing the closing `}` for the `function(asyncResult)`. Should be written like this:

```
Office.context.mailbox.makeEwsRequestAsync(
    request_str,
    function(asyncResult) {
        if (asyncResult.status === Office.AsyncResultStatus.Succeeded) {
            console.log(asyncResult.value);
        }
        else {
            console.log(JSON.stringify(asyncResult));
        }
    }
);
```

File: docs/add-ins/metadata-for-an-outlook-add-in.md [hunk 2]
```diff
@@ -201,6 +201,57 @@ Custom properties set by an add-in are not equivalent to normal MAPI-based prope
...
+
+You can also get more custom properties if you specify them in the request string as other [**ExtendedFieldURI** elements](/exchange/client-developer/web-service-reference/extendedfielduri).
```
Comment (kbrandl) [inline docs/add-ins/metadata-for-an-outlook-add-in.md]: suggest that you not include the word "elements" as part of the hyperlink. i.e., just link the name of the element:

"...if you specify them in the request string as other [**ExtendedFieldURI**](/exchange/client-developer/web-service-reference/extendedfielduri) elements."

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[6db63f3](https://github.com/OfficeDev/outlook-dev-docs/pull/465/commits/6db63f30cfafdf5fc77c0d6de70e75a662dbf8c0)_: 

:clock10: **Preparing**: average preparing time is _41 sec(s)_

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[6db63f3](https://github.com/OfficeDev/outlook-dev-docs/pull/465/commits/6db63f30cfafdf5fc77c0d6de70e75a662dbf8c0)_: 

:clock10: **Incremental building**: average incremental building time is _17 sec(s)_

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[6db63f3](https://github.com/OfficeDev/outlook-dev-docs/pull/465/commits/6db63f30cfafdf5fc77c0d6de70e75a662dbf8c0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/metadata-for-an-outlook-add-in.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-add-ews-example/docs/add-ins/metadata-for-an-outlook-add-in.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C20%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201812200017025368-465%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=djYSyBE2k6kfCrY40CPtOlP%2BWUvbHGSqBkUlUd7jOuI%3D&st=2018-12-20T00%3A13%3A03Z&se=2019-01-20T00%3A18%3A03Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**